UIKitVK/UIKitVK/Model/News/News.swift

```diff
@@ -0,0 +1,36 @@
+// News.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+// Тип поста
+enum NewsItemType: Decodable {
+    case post
+    case photo
+}
+
+// Содержание новости
+final class NewsItem: Codable {
+    var text: String
```

Comment:
```suggestion
    let text: String
```
может нам подойдут везде  let ?